data-prepper-logstash-configuration/src/test/java/org/opensearch/dataprepper/logstash/mapping/OpenSearchPluginAttributesMapperTest.java

```diff
@@ -39,47 +36,6 @@ private OpenSearchPluginAttributesMapper createObjectUnderTest() {
         return new OpenSearchPluginAttributesMapper();
     }
 
-    @Test
```

Comment:
I don't think we should remove these tests. There should be a decent behavior even if there is no `defaultSettings` provided.